src/threadnetwork.cs

```diff
@@ -0,0 +1,74 @@
+using CoreFoundation;
+using ObjCRuntime;
+using Foundation;
+
+using System;
+
+namespace ThreadNetwork {
+
+	[iOS (15,0)]
+	[BaseType (typeof (NSObject))]
+	interface THClient
+	{
+		[Async]
+		[Export ("retrieveAllCredentials:")]
+		void RetrieveAllCredentials (Action<NSSet<THCredentials>, NSError> completion);
+
+		[Async]
+		[Export ("deleteCredentialsForBorderAgent:completion:")]
+		void DeleteCredentialsForBorderAgent (NSData borderAgentId, Action<NSError> completion);
+
+		[Async]
+		[Export ("retrieveCredentialsForBorderAgent:completion:")]
+		void RetrieveCredentialsForBorderAgent (NSData borderAgentId, Action<THCredentials, NSError> completion);
+
+		[Async]
+		[Export ("storeCredentialsForBorderAgent:activeOperationalDataSet:completion:")]
+		void StoreCredentialsForBorderAgent (NSData borderAgentId, NSData activeOperationalDataSet, Action<NSError> completion);
```

Comment:
remove `ForBorderAgent` since it's the first parameter, not the action

Reply:
same.